packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -900,6 +937,107 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+/// This example shows a [Scaffold] with an [AppBar], a [BottomAppBar] and six
+/// [FloatingActionButton]s, each in a different [FloatingActionButtonLocation].
+/// Each [FloatingActionButton] is configured using a [FloatingActionButtonConfiguration]
+/// and has a different callback.
+///
+/// ```dart
+/// int _location = 0;
+///
+/// static const List<FloatingActionButtonLocation> _Locations = [
+///   FloatingActionButtonLocation.endTop,
+///   FloatingActionButtonLocation.centerTop,
+///   FloatingActionButtonLocation.startTop,
+///   FloatingActionButtonLocation.startFloat,
+///   FloatingActionButtonLocation.centerFloat,
+///   FloatingActionButtonLocation.endFloat,
+/// ];
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   return Scaffold(
+///     appBar: AppBar(
+///       title: Text('Flutter Demo'),
+///     ),
+///     body: Center(
+///       child: RaisedButton(
+///         child: Text('Switch'),
+///         onPressed: () {
+///           setState(() {
+///             _location ++;
+///             print('Switch-button pressed.');
+///           });
+///         },
+///       ),
+///     ),
+///     floatingActionButton: FloatingActionButton(
+///       onPressed: () {print('add');},
+///       tooltip: 'Increment',
+///       child: Icon(Icons.add),
+///       backgroundColor: Colors.blue,
+///     ),
+///     floatingActionButtonLocation: _Locations[(_location + 0)%6],
+///     additionalFloatingActionButtonConfigurations: {
+///       ValueKey('beach_access_button'): FloatingActionButtonConfiguration(
+///         button: FloatingActionButton(
+///           onPressed: () {
+///             print('beach_access');
+///           },
+///           child: Icon(Icons.beach_access),
+///           backgroundColor: Colors.yellow,
+///           foregroundColor: Colors.black,
+///         ),
+///         location: _Locations[(_location + 1) % 6],
```

Comment:
nit (here and below):

```suggestion
///         location: _locations[(_location + 1) % 6],
```

Reply:
Done.